### ISSUE TITLE
consider merging with jshint-rb

### ISSUE THREAD

**Issue by josephholsten** [Bgouveia/jshint_on_rails#7]
I'm trying to start maintaining a fork of jshint_on_rails at https://github.com/josephholsten/jshint.rb

It's mostly this repo rebased onto the latest jslint_on_rails

I'm not really attached to the implementation, I'd just like to make sure there is at least one well-maintained jshint to ruby adapter.

**Comment by Bgouveia** [comment]
Hi Joseph, I agree with you. Lately I'm not very dedicated to reviewing, testing and merging contributions to this project (sorry =S). As I saw (from your repo) everything is pretty much the same, and you actually did some stuff that I would've done (congratz =D). You seem to be a lot more active than I, and I think it is very healthy to this project to:
1) transfer the rubygems reference to your repo or;
2) make you a collaborator so you can have full access. 
What do you think? 
We have a lot of implementations, I appreciate your move trying to improve the project maintainability.

**Comment by josephholsten** [comment]
Btw, you've got nothing to be sorry about! I'd have had to do so much more work without you!

My main concerns are the wide number of disparate effort and implementations, followed closely by the need to stay on top of upstream changes.

I'm not especially attached to my repo, so it would probably be ideal for us to maintain the brand of jshint_on_rails and drop my fork. So a commit bit and access to push the gem should be all I need. 

There's a bunch of other jshint wrappers out there as well, I've started to approach those folks to see if we can all work together. 

## 

http://josephholsten.com

**Comment by Bgouveia** [comment]
I agree, but maybe we could create an organization to help us all to manage this project.

Could you make a pull request to my repo? I'll accept it as soon as you make it and push it to v1.1.0

**Comment by jaredbeck** [comment]
Any progress on this?  Looking forward to using a modern jshint version in my CI.  Thanks for your efforts!

**Comment by enriquecastl** [comment]
Looking forward to collaborate on this guys. Anything I can help with.

**Comment by Bgouveia** [comment]
Still merging repos, probably friday this'll be out. =)

**Comment by jaredbeck** [comment]
@josephholsten Any progress on this?  I'm trying out `jshint-rb 1.1.1` right now, but it looks like there's at least one open issue that needs addressing: https://github.com/josephholsten/jshint.rb/issues/4
